workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/ReadersWriters/Delegates.cs

```diff
@@ -1,5 +1,8 @@
-﻿using Improbable.Worker;
+﻿using System;
```

Comment:
Nit: File name is not equal to the class name.

Reply:
Good eye!